src/dav.js

```diff
@@ -15,6 +15,7 @@ export class Dav {
     }
 
     this.baseUrl = baseUrl
+    this.baseUrlv2 = baseUrlv2
```

Comment:
should we then call `baseUrl` `baseUrlv1` ?

Reply:
I would say it's consistent this way, since you also have `client` and `clientv2`. I can change it, but it would need to change in many places.

Reply:
also in favour of keeping `baseUrl` for the first one.